doc/content/concepts/devicetemplates/assigning-euis.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "Assigning EUIs"
+description: ""
+weight: 6
+---
+
+You can use the `assign-euis` command to assign the `JoinEUI` and `DevEUI` to device templates. The `DevEUI` is assigned from the given start number. This personalizes the device templates.
```

Comment:
Add full command for consistency.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/1016/commits/8fcc55545ece18e76b6c26611283ede09eab9afc